index-fr.html

```diff
@@ -42,7 +42,7 @@
     <div class="overlay">
       <div>
         <h1>Le pouvoir multiplicateur des masques</h1>
-        <h3>Un essai interactif sur la manière dont les masques peuvent mettre un terme à la COVID-19</h3>
+        <h3>Un essai interactif sur la manière dont les masques peuvent mettre un terme à COVID-19</h3>
```

Comment:
"au COVID-19" ("à COVID-19" is not correct for either masculine or feminine gender of COVID-19).